tests/mitmproxy_addons/callback.py

```diff
@@ -83,25 +124,34 @@ async def response(self, flow):
                 "request_body": req_body,
                 "response_body": res_body,
             }
-            try:
-                # use asyncio so we don't block other unrelated requests from being processed
-                async with aiohttp.request(
-                    method="POST",url=self.config["callback_url"], timeout=aiohttp.ClientTimeout(total=10),
-                    headers={"Content-Type": "application/json"},
-                    json=callback_body) as response:
-                    print(f'{datetime.now().strftime("%H:%M:%S.%f")} callback for {flow.request.url} returned HTTP {response.status}')
-                    test_response_body = await response.json()
-                    # if the response includes some keys then we are modifying the response on a per-key basis.
-                    if len(test_response_body) > 0:
-                        respond_status_code = test_response_body.get("respond_status_code", flow.response.status_code)
-                        respond_body = test_response_body.get("respond_body", res_body)
-                        flow.response = Response.make(
-                            respond_status_code, json.dumps(respond_body),
-                            headers={
-                                "MITM-Proxy": "yes", # so we don't reprocess this
-                                "Content-Type": "application/json",
-                            })
+            await self.send_callback(flow, self.config["callback_response_url"], callback_body)
 
-            except Exception as error:
-                print(f"ERR: callback for {flow.request.url} returned {error}")
-                print(f"ERR: callback, provided request body was {callback_body}")
+    async def send_callback(self, flow, url: str, body: dict):
+        try:
+            # use asyncio so we don't block other unrelated requests from being processed
+            async with aiohttp.request(
+                method="POST",url=url, timeout=aiohttp.ClientTimeout(total=10),
+                headers={"Content-Type": "application/json"},
+                json=body) as response:
```

Comment:
This is a bit illegible

```suggestion
            async with aiohttp.request(
                method="POST",
                url=url,
                timeout=aiohttp.ClientTimeout(total=10),
                headers={"Content-Type": "application/json"},
                json=body,
            ) as response:
```

Reply:
btw I heartily recommend getting some [Black](https://github.com/psf/black) in your python life

Reply:
I didn't know about black, I'll add that in another PR, thanks!